.eslintrc

```diff
@@ -18,6 +18,7 @@
   "rules": {
     "comma-dangle": [2,"never"],
     "linebreak-style": ["error", "unix"],
+    "no-console": "off",
```

Comment:
Do you think it would be worthwhile to bring in a logging library? I have used loglevel.js, https://www.npmjs.com/package/loglevel, in the past.

Reply:
Yes that looks like a good library. In this case, I was envisioning using it just to notify us during development of errors.... I had an exception that was being silenced by a promise and needed to see its contents. But seems we should consider a comprehensive approach.